registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -647,6 +650,7 @@
     "sha256:0a1cbd496e14bf620c4ae6bba3b8e63560bc116d8def7c85bd7733c02f789e2e": ["v1.11.0-beta.0"]
     "sha256:09bf2770f060110815aa20f5e9bebf719de0489d2f2d33e2a17168a47be10ed0": ["v1.11.0-beta.1"]
     "sha256:125840477545e10c1c9d3e4e688dac234274385f57a7b2ca71e225157fc78457": ["v1.11.0-beta.2"]
+    "sha256:ee1dff03202fdab82477fd66e0bde37115f8b7e44fe07bc430f872bf4fbfe4a4": ["v1.11.0-rc.0"]
```

Comment:
SHA `ee1dff03202f` matches kubeadm-control-plane-controller v1.11.0-rc.0 in the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-control-plane-controller@sha256:ee1dff03202fdab82477fd66e0bde37115f8b7e44fe07bc430f872bf4fbfe4a4/details?inv=1&invt=Ab4ryw).